### ISSUE TITLE
Review design system website against new typography scale

### ISSUE THREAD

**Issue by owenatgov** [alphagov/govuk-design-system#2862]
## What
Review the design system website against the new typography scale, checking for potential "breakages" that need to be fixed eg: where the new typography negatively impacts page hierarchy.

## Why
To Ensure our website is still usable and accessible ahed of publishing the new typography.

To "test" the new typography in a real service and collect data on the testing experience.

Part of https://github.com/alphagov/govuk-design-system/issues/2289. Requires https://github.com/alphagov/govuk-design-system/issues/2299 to be finished before proceeding.

## Who needs to work on this
Designers, developers

## Who needs to review this
Team members working on the typography epic

## Done when

- [ ] Establish a methodology for testing the website eg: by page? by unique visual element?
- [ ] Run through initial test
- [ ] Discuss actions to take, if any

**Comment by owenatgov** [comment]
# Result of testing
From running through the website using https://github.com/alphagov/govuk-design-system/pull/3249, my take is that there are no major issues 🎉 However, I think there are things we can investigate and try to fix.

## Space between lists
Currently, lists have a 5px space between them by default which is removed for bulleted and numbered lists. This spacing is unified from tablet onwards. This was implemented as part of https://github.com/alphagov/govuk-frontend/issues/1069. The intent of this issue was to increase space between non-bullet non-numbered lists, but the decision was made to maintain the lack of spacing on mobile on the basis that the bullet and numbers indicate the beginning of a new line.

I personally think it's worth re-evaluating this for the new scale as lists now feel squashed. This can be seen on the [button component page](https://design-system.service.gov.uk/components/button/). Here is the bullet list on mobile under the "how it works" heading at time of writing:
![Screenshot 2023-10-20 at 10 37 07](https://github.com/alphagov/govuk-design-system/assets/64783893/b92702cf-ab1d-4e57-a1ed-67191130040f)

...and here it is [against the new scale](https://deploy-preview-3249--govuk-design-system-preview.netlify.app/components/button/):
![Screenshot 2023-10-20 at 10 36 48](https://github.com/alphagov/govuk-design-system/assets/64783893/d55594b0-e640-4134-afc2-8af928f804e8)

I'll confer with designers on the team to figure out how best to handle this. My suggestion would be to review spacing for lists wholesale and then include these changes in the `type-scale` branch for deployment alongside the new scale.

## Use of 14 on the typography scale in the website
The new tab link in our example partial was using 14 on the typography scale and ideally we should be consistent with not using 14 on the font scale in anticipation of removing it. This will be solved with https://github.com/alphagov/govuk-design-system/pull/3254.

## Documentation of 14 on the typography scale
We reference 14 as an option for typography in 2 places:

1. [Font override classes on the typography page](https://design-system.service.gov.uk/styles/typography/#font-size)
2. [The typography section of the upgrade to govuk-frontend page](https://design-system.service.gov.uk/get-started/updating-your-code/#typography)

I'm more confident after discussing this further with the team about 1, I'm less so about 2.

## Table on cookie page breaking
The first table where we list the single essential cookie on the [cookie page](https://design-system.service.gov.uk/cookies/) overflows when you get to roughly 400px screen width. This is exacerbated by the new scale where it now breaks at 450px width. This requires further investigation to see if we can implement word breaking. I suspect it will involve a change to the table component.

## Examples without iframe missing borders
When an example doesn't have an iframe showing the code in situ, on mobile it doesn't have a top border. This is a minor bug which has been fixed by https://github.com/alphagov/govuk-design-system/pull/3253.

## Spacing on component list on small screens
The list of components on small screens on the [component landing page](https://deploy-preview-3249--govuk-design-system-preview.netlify.app/components/) is now a bit tight. Suggest increasing bottom margin to 15px.

**Comment by owenatgov** [comment]
Issues made off of the above results:

- https://github.com/alphagov/govuk-frontend/issues/4374
- https://github.com/alphagov/govuk-design-system/issues/3265
- https://github.com/alphagov/govuk-design-system/issues/3258

Closing this as done.
